examples/oneapi/cpp/source/kmeans/kmeans_lloyd_dense_batch.cpp

```diff
@@ -35,14 +36,15 @@ int main(int argc, char const *argv[]) {
 
   const auto data_table = dal::homogen_table{row_count, column_count, data};
   const auto initial_centroids_table =
-      dal::homogen_table{1, column_count, initial_centroids};
+      dal::homogen_table{cluster_count, column_count, initial_centroids};
```

Comment:
😄 Why does the first option work? Is this a bug in the tables?

Reply:
cause `1 * column_count < cluster_count * column_count`

Reply:
logically in the first case there should not be a segfault?

Reply:
It really should be thrown from the inside the kernel. But there is no reasons to explore this case, because oneDAL provides such checks for users before the kernel.